pcweb/pages/docs/gallery.py

```diff
@@ -24,7 +24,7 @@
         "description": "An AI chat app.",
         "img": "/gallery/chat.gif",
         "gif": "",
-        "url": "https://webui-teal-star.reflex.run",
+        "url": "https://chat.reflex.run/",
```

Comment:
https://chat.reflex.run this goes the the old deployed site, I think the other one is the new deployment. Although we can deploy to the other URL as its more asthetic

Reply:
Oh I didn't mean to include the change to gallery.py in that PR, I'll remove that from here and move it to a separate PR.